games-arcade/savagewheels/savagewheels-1.6.1-r1.ebuild

```diff
@@ -62,19 +62,17 @@ src_configure() {
 	fi
 
 	mycmakeargs+=(
-		-DCMAKE_INSTALL_DATADIR=/usr/share/games/${PN}
-		-DCMAKE_INSTALL_DATAROOTDIR=/usr/share
-		-DCMAKE_INSTALL_DOCDIR=/usr/share/doc/${P}
+		-DCMAKE_INSTALL_DATADIR=/usr/share/${PN}
 		-DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec/${PN}
-		-DCMAKE_INSTALL_PREFIX=/usr/games
+		-DCMAKE_INSTALL_PREFIX=/usr
 	)
 	cmake-utils_src_configure
 }
 
 src_install() {
 	cmake-utils_src_install
 
-	insinto "/usr/share/games/${PN}"
+	insinto "/usr/share/${PN}"
 	for d in "${WORKDIR}/${GAMEDATA}/*"; do
 		doins -r ${d}
```

Comment:
A bit unrelated but can't stand seeing it. I think the following would work:

    doins -r "${WORKDIR}/${GAMEDATA}/."

(instead of the whole loop)